geoengine/workflow.py

```diff
@@ -342,20 +342,21 @@ def __request_wcs(
 
         [resx, resy] = bbox.resolution_ogc
 
-        no_data_value = ""
+        kwargs = {}
+
         if force_no_data_value is not None:
-            no_data_value = str(float(force_no_data_value))
+            kwargs["no_data_value"] = str(float(force_no_data_value))
 
         return wcs.getCoverage(
             identifier=f'{self.__workflow_id}',
             bbox=bbox.bbox_ogc,
-            time=[urllib.parse.quote_plus(bbox.time_str)],
+            time=[bbox.time_str],
             format=file_format,
             crs=crs,
             resx=resx,
             resy=resy,
             timeout=timeout,
-            nodatavalue=no_data_value,
+            **kwargs
```

Comment:
It seems like the value changed from `nodatavalue` to `no_data_value`. Why didn't you change just the left hand side?

Reply:
https://github.com/geo-engine/geoengine-python/pull/131/commits/07d32f63e96ce438b96b72a09f112050a1ebcbda